service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RespondentMidEventControllerTest.java

```diff
@@ -72,6 +75,41 @@ void shouldReturnNoErrorsWhenAllDatesOfBirthAreTodayOrInPast() throws Exception
         assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
     }
 
+    @Test
+    void shouldReturnErrorsWhenThereIsNewRespondentAndNoOldRespondent() throws Exception {
```

Comment:
The test says `NewRespondentAndNoOldRespondent` while only new respondent structure is populated.

Also there is no test for happy scenario. Having this done well now will allow us just to remove old 3 test cases when old structure is removed.